### PR TITLE
🎉 New Source: DataScope [low-code cdk]

### PR DESCRIPTION
**Pull request by jhammarstedt**
## New Connectors
Adding Datascope connector with low code CDK

## Recommended reading order
1. `x.java`
2. `y.python`

## 🚨 User Impact 🚨
Are there any breaking changes? What is the end result perceived by the user? If yes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- [X ] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [X ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [X ] Documentation updated
    - [X ] Connector's `README.md`
    - [x] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [X] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [X] `docs/integrations/README.md`
    - [X] `airbyte-integrations/builds.md`
- [X] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

</details>

## Tests

<details><summary><strong>Integration</strong></summary>

</details>

<details><summary><strong>Acceptance</strong></summary>

image

</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-datascope/source_datascope/datascope.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    $options:
+      name: "locations"
+      primary_key: "id"
+      path: "/locations"
+  answers_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "answers"
+      primary_key: "form_answer_id"
+      path: "/v2/answers"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-datascope/source_datascope/datascope.yaml:45]: @jhammarstedt Datascope API allows start/end parameter which enables incremental syncs. This feature is very useful to users, what do you think about it? Let me know if you need any help to implement.

Comment (marcosmarxm) [reply airbyte-integrations/connectors/source-datascope/source_datascope/datascope.yaml:45]: It is the same for notifications endpoint.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
The connector is working with full refresh. Asking if you are interested in implementing incremental syncs.

**Comment by marcosmarxm** [comment]
Hello! I'm going to be out of the office this Friday and won't be able to review your contribution again today, I return next Monday. So far, most contributions look solid and are almost done to be approved. As said in [Chris' comment](https://airbytehq.slack.com/archives/C0450PCSDB2/p1667430678866629) all contributions made before 2-November are eligible to receive the prize and have 2 weeks to merge the contributions. But I ensure next week we're going to have your contribution merged. If you have questions about the implementation you can send them in `#hacktoberfest-2022` in Airbyte's Slack.

Sorry the inconvenience and see you again next week, thank you so much for your contribution!

File: airbyte-integrations/connectors/source-datascope/source_datascope/datascope.yaml (hunk 1 above)
Comment (jhammarstedt) [reply airbyte-integrations/connectors/source-datascope/source_datascope/datascope.yaml:45]: Hi, sure I can give incremental a go! I'll see if I can get it done over the weekend!

**Comment by jhammarstedt** [comment]
status: I've added support for incremental syncs (will push it up today). Had some issue with the acceptance test there but I've probably missed some config, might ask for some help/clarification on that later

**Comment by marcosmarxm** [comment]
@jhammarstedt please ping me when this is done!

**Comment by jhammarstedt** [comment]
@marcosmarxm Added incremental (works fine to run but having some issues with the acceptance test)

File: airbyte-integrations/connectors/source-datascope/source_datascope/datascope.yaml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+  location_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "locations"
+      primary_key: "id"
+      path: "/locations"
+      stream_cursor_field: "date"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-datascope/source_datascope/datascope.yaml:58]: locations doesn't have date field value.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Please check all streams have `date` or a cursor field.

File: airbyte-integrations/connectors/source-datascope/source_datascope/datascope.yaml (hunk 2 above)
Comment (jhammarstedt) [reply airbyte-integrations/connectors/source-datascope/source_datascope/datascope.yaml]: @marcosmarxm resolved!

**Comment by jhammarstedt** [comment]
@marcosmarxm Removed and updated all non-relevant fields.

**Review by marcosmarxm — APPROVED** [review]
Thanks @jhammarstedt

**Pull request merged**